include/zephyr/bluetooth/mesh/brg_cfg_cli.h

```diff
@@ -0,0 +1,313 @@
+
+/** @brief Sends a Bridging Table Add message to the given destination address
+ *  with the given parameters
+ *
+ *  This function sends a Bridging Table Add message to the given destination
+ *  address with the given parameters to add an entry to the Bridging Table. The
+ *  Bridging Table contains the net keys and addresses that are authorized to be
+ *  bridged by the node. The function expects a Bridging Table Status message as
+ *  a response from the destination node.
+ *
+ *  This method can be used asynchronously by setting @p rsp as NULL. This way
+ *  the method will not wait for response and will return immediately after
+ *  sending the command.
+ *
+ *  @param net_idx    Network index to encrypt the message with.
+ *  @param addr       Target node address.
+ *  @param directions Allowed directions for the bridged traffic
+ *  @param net_idx_1  NetKey Index of the first subnet
+ *  @param net_idx_2  NetKey Index of the second subnet
+ *  @param addr_1     Address of the node in the first subnet
+ *  @param addr_2     Address of the node in the second subnet
+ *  @param rsp        Status response parameter
+ *
+ *  @return 0 on success, or (negative) error code on failure.
+ */
+int bt_mesh_brg_cfg_cli_bridging_table_add(uint16_t net_idx, uint16_t addr, uint8_t directions,
+					   uint16_t net_idx_1, uint16_t net_idx_2, uint16_t addr_1,
+					   uint16_t addr_2,
+					   struct bt_mesh_bridging_table_status *rsp);
+
+/** @brief Sends a Bridging Table Remove message to the given destination
+ *  address with the given parameters
+ *
+ *  This function sends a Bridging Table Remove message to the given destination
+ *  address with the given parameters to remove an entry from the Bridging
+ *  Table. The Bridging Table contains the net keys and addresses that are
+ *  authorized to be bridged by the node. The function expects a Bridging Table
+ *  Status message as a response from the destination node.
+ *
+ *  This method can be used asynchronously by setting @p rsp as NULL. This way
+ *  the method will not wait for response and will return immediately after
+ *  sending the command.
+ *
+ *  @param net_idx   Network index to encrypt the message with.
+ *  @param addr      Target node address.
+ *  @param net_idx_1 NetKey Index of the first subnet
+ *  @param net_idx_2 NetKey Index of the second subnet
+ *  @param addr_1    Address of the node in the first subnet
+ *  @param addr_2    Address of the node in the second subnet
+ *  @param rsp       Pointer to a struct storing the received response from the
+ *                   server, or NULL to not wait for a response.
+ *
+ *  @return 0 on success, or (negative) error code on failure.
+ */
+int bt_mesh_brg_cfg_cli_bridging_table_remove(uint16_t net_idx, uint16_t addr, uint16_t net_idx_1,
+					      uint16_t net_idx_2, uint16_t addr_1, uint16_t addr_2,
+					      struct bt_mesh_bridging_table_status *rsp);
```

Comment:
It may be easier to use this in the code, if we drop `_` from config params.

```suggestion
int bt_mesh_brg_cfg_cli_bridging_table_remove(uint16_t net_idx, uint16_t addr, uint16_t netidx1,
					      uint16_t netidx2, uint16_t addr1, uint16_t addr2,
					      struct bt_mesh_bridging_table_status *rsp);
```

or:`net_idx1` and `addr1` ...

Reply:
I think when the _ is there it is more readable, and I think that is important when there is many input variables with the similar names. Other than that I do not have strong opinion on this, but if we should remove the _ I think we should compact it as much as possible.

Reply:
Zephyr's mesh doesn't use such an approach. It should be `net_idx1` and `addr1`.